modules/form/src/addons/index.ts

```diff
@@ -17,3 +17,4 @@
  */
 
 export * from "./query-params";
+export * from "./scopes";
```

Comment:
Fix the lint errors. Check in other places as well

Reply:
resolved with c06e2cbf6d9681e4816f61839b6565d7ec46776f